recipes/rich/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "rich" %}
+{% set version = "0.8.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c783f1c99cc709992e6a1cf9f0d04fc27c205a8920a87d1e24ced0aab49be26f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
+
+requirements:
+  host:
+    - python
+    - pip
+    - poetry
+  run:
+    - python
+
+test:
+  imports:
+    - rich
```

Comment:
add `- rich.console` since just importing the top level module doesn't do anything and could miss dependencies